docs/swagger.yaml

```diff
@@ -822,6 +927,10 @@ components:
         email:
           type: string
           description: 'Example: "user@example.com"'
+        userInfoSub:
```

Comment:
Should remove this for now - it is hidden for a while to assess how it works for KomplettNettbutikk.